test/e2e/tests/resize_e2e_test.go

```diff
@@ -39,6 +41,7 @@ const (
 	defaultTier          = "STANDARD"
 	defaultNetwork       = "default"
 	minVolumeSize  int64 = 1 * util.Tb
+	defaultEpsilon int64 = 70 * util.Gb // when a 1T instance was created, the actual size is 1007G, and only 956G is available to non-root users, so there will be ~70 Gb of space unavailable
```

Comment:
since we changed how the free space is reported, we can change this epsilon to 20G

Reply:
Fixed